pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -312,7 +312,7 @@ void processArguments() throws Exception {
         @Parameter(names = "--autoAck", description = "Whether or not the framework will automatically acknowleges messages", hidden = true)
         protected Boolean DEPRECATED_autoAck = null;
         @Parameter(names = "--auto-ack", description = "Whether or not the framework will automatically acknowleges messages")
-        protected Boolean autoAck ;
+        protected boolean autoAck ;
```

Comment:
@rdhabalia With JCommander shouldn't it be able to handle boolean flags with default value to `true` ?

Eg. with : 

```java
protected boolean autoAck = true;
```

shouldn't be working to pass `--auto-ack=false`  ?

Reply:
http://jcommander.org/#_boolean